lighthouse-core/gather/driver.js

```diff
@@ -59,6 +59,10 @@ const DEFAULT_PROTOCOL_TIMEOUT = 30000;
  */
 
 class Driver {
+  online = true;
+  // eslint-disable-next-line no-invalid-this
+  fetcher = new Fetcher(this);
+
```

Comment:
why not move all the other fields at the same time? Can mark them as `@private` to sidestep the `#` question for now :)

Reply:
Done, but... 
![image](https://user-images.githubusercontent.com/4071474/98877594-712cbc80-2446-11eb-9214-43b29d54a63c.png)
Thoughts?

Reply:
> Thoughts? Drop the private marker?

Ah, yeah, that's unfortunate. Of course in theory we shouldn't be touching private state for tests, but obviously that's often not possible or makes things worse. I guess no `@private` :/

Reply:
I marked it.... `@pri_vate`.

Reply:
> why not move all the other fields at the same time?

too late now, but a good reason IMO would have been that it creates merge conflicts for in progress PRs and we only needed to do 1 small node 12 syntax feature for our v7 goals :)